retriever/lib/engine.py

```diff
@@ -385,9 +387,10 @@ def database_name(self, name=None):
             db_name = name
         return db_name.replace('-', '_')
 
-    def download_file(self, url, filename, use_cache=True):
+    def download_file(self, url, filename):
         """Downloads a file to the raw data directory."""
-        if not self.find_file(filename) or not use_cache:
+        if not self.find_file(filename) or (not self.use_cache and not self.downloaded):
+            self.downloaded = True
```

Comment:
I think it would be better to add it in both the ```try``` and ```except``` statements. This way it will be more understandable.